02_Architecture/07_APIC.md

```diff
@@ -11,13 +11,13 @@ It's worth noting that Intel later developed a version of the APIC called the SA
 There are two types of APIC:
 
 * _Local APIC_: it is present in every processor core, it is responsible for handling incoming interrupts for *that core*. It can also be used for sending an IPI (inter-processor interrupt) to other cores, as well as generating some interrupts itself. Interrupts generated by the local APIC are controlled by the LVT (local vector table), which is part of the local APIC registers. The most interesting of these is the timer LVT, which we will take a closer look at in the timers chapter.
-* _IO/APIC_: An IO APIC acts a 'gateway' for devices in the system to send interrupts to local APICs. Most personal computers will only have a single IO APIC, but more complex systems (like servers or industrial equipment) may contain multiple IO APICs. Each IO APIC has a number of input pins, which a connected device triggers when it wants to send an interrupt. When this pin is triggered, the IO APIC will send an interrupt to one (or many) local APICs, depending on the *redirection entry* for that pin. We can program these redirection entries, and they're presented as an array of memory-mapped registers. We'll look more at this later. In summary, an IO APIC allows us to route device interrupts to processor cores however we want.
+* _I/O APIC_: An I/O APIC acts a 'gateway' for devices in the system to send interrupts to local APICs. Most personal computers will only have a single I/O APIC, but more complex systems (like servers or industrial equipment) may contain multiple I/O APICs. Each I/O APIC has a number of input pins, which a connected device triggers when it wants to send an interrupt. When this pin is triggered, the I/O APIC will send an interrupt to one (or many) local APICs, depending on the *redirection entry* for that pin. We can program these redirection entries, and they're presented as an array of memory-mapped registers. We'll look more at this later. In summary, an I/O APIC allows us to route device interrupts to processor cores however we want.
 
 Both types of APIC are accessed by memory mapped registers, with 32-bit wide registers. They both have well-known base addresses, but rather than hardcoding these they should be fetched from the proper places as firmware (or even the bootloader) may move these around before our kernel boots.
 
 ## Local APIC
 
-When a system boots up, the cpu starts in PIC8259A emulation mode for legacy reasons. This simply means that instead of having the LAPIC/IO-APIC up and running, we have them working to emulate the old interrupt controller, so before we can use them properly we should to disable the PIC8259 emulation.
+When a system boots up, the cpu starts in PIC8259A emulation mode for legacy reasons. This simply means that instead of having the LAPIC/I/O APIC up and running, we have them working to emulate the old interrupt controller, so before we can use them properly we should to disable the PIC8259 emulation.
```

Comment:
I think that for can be confusing LAPIC/I/O APIC since there is also the extra / from. Maybe using LAPIC-I/O APIC or maybe directly LAPIC or I/O APIC? (or anything similar)

Reply:
I chose "LAPIC and I/O APIC"